Templates/JenkinsPipeline/Jenkinsfile

```
@@ -591,28 +608,32 @@ pipeline {
 							// Locate the UCD Component Version URL in the output log.
 							def ucdVersionUrl = searchLogOutput("version.url -> (.*)", readFile("${ucdBuzToolLog}"))
 							if (ucdVersionUrl){
-								println("${PipelineName}[INFO]: UCD Component Version URL: ${ucdVersionUrl}")
-								createSummary icon:"accept.svg", text: "${MsgHdr} <a href=\'$ucdVersionUrl\' target=\'_other\'>UCD Component Version (name: $ucdVersionName)</a>"
+								Msg = "Package succesfully created."
+								println("${PipelineName}[INFO]:")
```

Missing to pass the Msg to println
```suggestion
								println("${PipelineName}[INFO]: ${Msg}")
```